docs/src/main/paradox/kafka.md

```diff
@@ -6,11 +6,12 @@ A typical source for Projections is messages from Kafka. Akka Projections has in
 The @apiddoc[KafkaSourceProvider] uses consumer group assignments from Kafka and can resume from offsets stored in
 a database.
 
-Akka Projections can store the offsets from Kafka in a @ref:[relational DB with Slick](slick.md).
+Akka Projections can store the offsets from Kafka in a @ref:[relational DB with JDBC](jdbc.md)
+or in @ref:[relational DB with Slick](slick.md).
 
-The `SlickProjection` envelope handler returns a `DBIO` that will be run by the projection. This means that the target database
-operations can be run in the same transaction as the storage of the offset, which means that @ref:[exactly-once](slick.md#exactly-once)
-processing semantics is supported. It also offers @ref:[at-least-once](slick.md#at-least-once) semantics.
+The `JdbcProjection` @scala[or `SlickProjection`] envelope handler will be run by the projection. This means that the target database
+operations can be run in the same transaction as the storage of the offset, which means that @ref:[exactly-once](jdbc.md#exactly-once)
```

Comment:
May also link to slick docs?

Reply:
Slick is mentioned at the top of the page, but otherwise we should promote JDBC mostly.